### PR TITLE
Added "Hello Martin" to server and handled response with js.

### PR DESCRIPTION
**Pull request by Bickor**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -29,6 +29,9 @@
 
   private ArrayList<String> messages;
```
Comment (tzsebe) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:30]: Use interfaces (`private List<String> messages;`).

File: walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/script.js [hunk 2]
```diff
@@ -20,7 +20,7 @@ function getRandomQuote() {
 
   // The fetch() function returns a Promise because the request is asynchronous.
   const responsePromise = fetch('/random-quote');
-
+    console.log(responsePromise);
```
Comment (mkmg) [inline walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/script.js:23]: If this was just for debugging purposes please remove.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 3]
```diff
@@ -20,13 +20,45 @@
...
+   * the Gson library dependency to pom.xml.
+   */
+  private String convertToJsonUsingGson(ArrayList messages) {
+    Gson gson = new Gson();
+    String json = gson.toJson(messages);
+    return json;
+  }
+
+  /**
+    * Function to give content to the fronend.
```
Comment (mkmg) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:54]: Nit: frontend

Comment (Bickor) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:54]: Done!

**Comment by mkmg** [comment]
In the future, keep a pull request open until it is approved and ready to merge. I sent out the developer workflow doc earlier today, so read through that and then let me know if you have questions about this.

**Comment by Bickor** [comment]
I apologize for the late reply, I just saw this on my personal email. At the time I didn't know how to ask for another request to review after suggested changes so my way of dealing with it was closing the pull request and making another similar pull request. I apologize for the inconvenience!
Now I know how to deal with it so I will follow the correct method in the future.

**Comment by mkmg** [comment]
No need to apologize; the workflow doc hadn't been published yet so there was no way for you to know how to do it. :)

**Pull request closed without merging**